src/qml/qgismobileapp.qml

```diff
@@ -337,23 +337,29 @@ ApplicationWindow {
 
     text: ( qfieldSettings.numericalDigitizingInformation && stateMachine.state === "digitize" ) || stateMachine.state === 'measure' ?
               '%1%2%3%4'
-                .arg(stateMachine.state === 'digitize' || !digitizingToolbar.isDigitizing ? '<p>%1 / %2</p>'
-                  .arg(coordinateLocator.currentCoordinate.x.toFixed(3))
-                  .arg(coordinateLocator.currentCoordinate.y.toFixed(3))
+                .arg(stateMachine.state === 'digitize' || !digitizingToolbar.isDigitizing ? '<p>%1: %2<br>%3: %4</p>'
+                  .arg(coordinateLocator.mapSettings.destinationCrs.isGeographic ? 'Lon' : 'X')
```

Comment:
Translate Lat/Lon/X/Y?

Reply:
Let's make lat lon translatable. I don't think there's a point in translating X and Y :)